plugin/pkg/scheduler/scheduler.go

```diff
@@ -222,7 +225,7 @@ func (sched *Scheduler) bind(assumed *v1.Pod, b *v1.Binding) error {
 	if err != nil {
```

Comment:
yup, still prefer to call FinishBinding just after bind as before. But we can not revert #48451 , we need to just log error if 'FinishBinding' failed.

Current fix did not get a chance to call 'FinishBinding' which will still leave assumed pod in cache, so the pod can not be re-scheduled. For the concern on race of #48451, I think that's fine; 'ForgetPod' is quick enough, and scheduler will correct it in next cycle if it happened.